### ISSUE TITLE
module "QtQuick.XmlListModel" is not installed

### ISSUE THREAD

**Issue by navarroaxel** [I-Dream-in-Code/kde-arch-update-plasmoid#40]
I'm getting this error after a full upgrade of Arch Linux:
```
Error loading QML file: file:///usr/share/plasma/plasmoids/org.kde.archUpdate/contents/ui/main.qml:28:34: Type CompactRepresentation unavailable
file:///usr/share/plasma/plasmoids/org.kde.archUpdate/contents/ui/CompactRepresentation.qml:73:5: Type FullRepresentation unavailable
file:///usr/share/plasma/plasmoids/org.kde.archUpdate/contents/ui/FullRepresentation.qml:5:1: module "QtQuick.XmlListModel" is not installed
```
And the widget can't load. I tried to reinstall the widget, but it does not work.

**Comment by Maryse47** [comment]
Do you have `qt5-xmlpatterns` package installed?

**Comment by navarroaxel** [comment]
No, that fixed my widget! But I never installed or removed that package. Do you know if Arch Linux removed that from KDE because is deprecated or something?

Maybe a clean orphan packages has removed it.

I think is not a dependency of this package because is part of the core of KDE, but it's not the case.

**Comment by Maryse47** [comment]
It may get cleaned out at some point in your system. The proper fix is to add it explicitly as dependency in `PKGBUILD` @I-Dream-in-Code . You may also remove `fakeroot` which is already covered by `pacman-contrib`.
